main.css

```diff
@@ -39,41 +40,48 @@
     }
 
     .carousel-navbar {
+     background: rgba(0, 0, 0, 0.6);
      position: absolute;
-     bottom: 620px;
-     z-index: 10;
+     left: 0;
+     top: 0;
```

Comment:
Better to stick this to the top then offset from the bottom (because the height of the carousel changes as you resize the window)